src/lib/config.ts

```diff
@@ -981,6 +981,24 @@ export class default_config {
      */
     bmarkweight = 100
 
+    /**
+     * General completions options - NB: options are set according to our internal completion source name - see - `src/completions/[name].ts` in the Tridactyl source.
```

Comment:
I should probably add an "experimental" warning here, because I'm still toying with totally rewriting completions to use config rather than object-oriented-bad-smells, and then this would probably move to an excmd -> setting mapping rather than completion source -> setting jobbie. Or maybe it wouldn't, since we'll still want the concept of completion sources so that different excmds can use the same ones.